iceoryx_utils/source/posix_wrapper/semaphore.cpp

```diff
@@ -286,5 +337,25 @@ bool Semaphore::isNamedSemaphore() noexcept
 {
     return m_isNamedSemaphore;
 }
+
+SemaphoreError Semaphore::errnoToEnum(const int errnoValue) const noexcept
```

Comment:
shouldnt we use exact-width types like uint32_t here?

Reply:
No since we get the errno (type int) has input argument here.